featurebyte/api/feature.py

```diff
@@ -1155,3 +1155,14 @@ def notnull(self) -> Feature:  # pylint: disable=missing-function-docstring
         class_method=_list_versions,
         instance_method=_list_versions_with_same_name,
     )
+
+    def update_namespace_description(self, description: str) -> None:
```

Comment:
Should we add make the type `Optional[str]`? Would be good if we also add `@typechecked` decorator.